doc/tutorial.txt

```diff
@@ -282,6 +282,45 @@ util_max: Integer. Can be specified at task level or phase level. Sets the max
 utilization value the task can have; which helps capping the task from
 scheduler point of view.
 
+*** NUMA memory binding
+
+* nodes: Array of Integer. Define the NUMA binding of the thread. 
+Default value is all NUMA nodes of the system. An example : "nodes" : [0, 2, 3]
+
+"nodes" Can be specified at task level or phase level. As an example, below
```

Comment:
This only affects memory, right?
Wonder if we'd better add this information to the keyword we use, so to be able to differentiate between, say, membind and bind (which affects affinity as well).

Reply:
I can change it to something like "membind_nodes"
Is it ok?